assets/app/features/upgrade/upgrade.module.js

```diff
@@ -2,5 +2,5 @@
     'use strict';
 
     angular
-        .module('crowbarApp.upgrade', []);
+        .module('crowbarApp.upgrade', ['ui.router', 'pascalprecht.translate', 'suseData']);
```

Comment:
`pascalprecht.translate` is the module where the `$translate` resource is contained.
You can check https://angular-translate.github.io/ or https://angular-translate.github.io/docs/#/guide for more details, or ask me if you have more questions. :)